README.md

```diff
@@ -143,11 +143,11 @@ There are also a variety of test programs built under the `tests` directory:
 
 To run one or more ciphers only once use `--mem-bench`, which is suitable for memory usage profiling:
 
-	tests/test_kex --mem-bench ntru
+	tests/test_kex --mem-bench FIXME
```

Comment:
We shouldn't leave FIXME. In fact, we should remove test_kex altogether.

Reply:
I'll remove that in issue: https://github.com/open-quantum-safe/liboqs/issues/372